ContextApp/src/main/java/com/github/migi_1/ContextApp/client/ClientWrapper.java

```diff
@@ -0,0 +1,112 @@
+package com.github.migi_1.ContextApp.client;
+
+import com.github.migi_1.ContextMessages.AccelerometerMessage;
+import java.io.IOException;
+
+import com.jme3.network.Client;
+import com.jme3.network.Network;
+import com.jme3.network.serializing.Serializer;
+
+/**
+ * A wrapper class for a com.jme3.network.client object.
+ */
+public class ClientWrapper {
+    
+	/** The default port on which servers are running. */
+	private static final int PORT = 4321;
+        /** The amount of times the client should restart before sending an error.*/
+        private static final int RESTART_ATTEMPTS = 10;
+	
+	private Client client;
+        private ClientState state;
+        
```

Comment:
private Client client; (line 20) appears to have a weird indentation. Please fix this.